gtest/src/manager.rs

```diff
@@ -219,6 +247,7 @@ pub(crate) struct ExtManager {
     pub(crate) opt_binaries: BTreeMap<CodeId, Vec<u8>>,
     pub(crate) meta_binaries: BTreeMap<CodeId, Vec<u8>>,
     pub(crate) dispatches: VecDeque<StoredDispatch>,
+    pub(crate) scheduled_dispatches: HashMap<u32, VecDeque<DelayedDispatch>>,
```

Comment:
no guarantees in order here: I'd go with vector, inserting dispatches randomly, cause it depends on hashes and rocksdb/sp_io impl on-chain
Also need your opinion @LouiseMedova